### DISCUSSION TITLE
Opposite of deny-lists

### DISCUSSION THREAD

**Discussion by storkeu** [microsoft/presidio#892] [Q&A] [answered]
Hello,

We have use cases where not all detected entities should be anonymized. That is, some concrete entities detected as <PERSON>,<LOCATION>, etc, should not be replaced.

It would be nice to have something similar to the deny-list, but with the opposite result, not performing the anonymization. Can the current solution address this requirement? Any workaround?

Thanks.

**Comment by omri374** [comment, marked as answer]
Hi @storkeu, we already have it implemented (https://github.com/microsoft/presidio/pull/865) will be part of the next release.

**Reply by omri374** [reply]
Release 2.2.29 (available on PyPI) contains the allow list capability. See tutorial here: https://microsoft.github.io/presidio/tutorial/13_allow_list/
